scripts/basic-workflow-tests.sh

```diff
@@ -186,6 +186,22 @@ test_train() {
     fi
 
     ilab model train "${TRAIN_ARGS[@]}"
+
+    if [ "$FULLTRAIN" -eq 1 ]; then
+
+        # TODO Only cuda for now
+        ilab config init --non-interactive --train-profile="${SCRIPTDIR}/test-data/train-profile-a10.yaml"
```

Comment:
I would prefer this go up in `test_init` where we set up the config

Reply:
but you can check what configuration the script is running in at that time.  I'd like the output of `test_config` to be "set up the config for this e2e run" at the beginning.  That is more aligned to a user workflow. They configure once for their environment and then run through the workflow.

Reply:
so like:

```
test_init() {
    task Initializing ilab

    if [ "$FULLTRAIN" -eq 1 ]; then
        # TODO Only cuda for now
        ilab config init --non-interactive --train-profile="${SCRIPTDIR}/test-data/train-profile-a10.yaml"
    else
        [ -f config.yaml ] || ilab config init --non-interactive
    fi
```

Reply:
added this and the removed the config inits from the train test specifically

Reply:
yeah, that looks good